lib/ansible/modules/network/nxos/nxos_vpc.py

```diff
@@ -101,6 +101,18 @@
     pkl_src: 10.1.100.2
     pkl_dest: 192.168.100.4
     auto_recovery: true
+
+- name: Configure VPC
+  nxos_vpc:
+    domain: 10
+    role_priority: 28672
+    system_priority: 2000
+    delay_restore: 180
+    peer_gw: true
+    pkl_src: 1.1.1.2
+    pkl_dest: 1.1.1.1
+    pkl_vrf: vpckeepalive
+    auto_recovery: true
```

Comment:
@smolz I see you have another PR opened for this https://github.com/ansible/ansible/pull/41049 that has another param. 
Can you please merge these examples in to one example in a PR?

Reply:
Done!
https://github.com/ansible/ansible/pull/41111